tcms/testplans/tests/test_admin.py

```diff
@@ -40,3 +40,34 @@ def test_users_can_delete_testplan(self):
         )
         self.assertRedirects(response, reverse("core-views-index"))
         self.assertEqual(TestPlan.objects.filter(pk=self.test_plan.pk).exists(), False)
+
+    def test_history_display_testplan(self):
+        with self.subTest("Test History before Changes"):
+            response = self.client.get(
+                "/admin/testplans/testplan/%d/history/" % self.test_plan.pk
+            )
+            self.assertContains(response, "<pre></pre>")
+            self.assertContains(response, _("<td>Created</td>"))
+
+        old_test_plan_name = self.test_plan.name
+
+        with self.subTest("Test changes included in history"):
+            self.test_plan.name = "Test Demo"
+            self.test_plan.save()
+            response = self.client.get(
+                "/admin/testplans/testplan/%d/history/" % self.test_plan.pk
+            )
+            self.assertContains(
+                response,
+                f"<pre>--- name\n+++ name\n@@ -1 +1 @@\n-{old_test_plan_name}\n+Test Demo</pre>",
+            )
+
+        with self.subTest("Test Reverting Redirect to Object"):
+            response = self.client.post(
+                "/admin/testplans/testplan/%d/history/1/" % self.test_plan.pk,
+                follow=True,
+            )
+            self.assertRedirects(
+                response,
+                self.test_plan._get_absolute_url(),
```

Comment:
This method is dynamic, it evaluates the URL based on the latest name (e.g. after the save).  Grab the URL before you make any changes and reversals.

Reply:
`_get_absolute_url()` -> `get_absolute_url()` to make pylint happy.